deploy/playbooks/examples/deploy_vagrant.yml

```diff
@@ -1,33 +1,17 @@
 ---
-
-# install python2.7 on xenial nodes
```

Comment:
I don't think that'll work on a fresh xenial vm though would it? IIRC, you need to install python2.7 first before ansible can run any modules.